components/style/properties/gecko.mako.rs

```diff
@@ -1204,6 +1205,22 @@ fn static_assert() {
     pub fn copy_transform_from(&mut self, other: &Self) {
         unsafe { self.gecko.mSpecifiedTransform.set(&other.gecko.mSpecifiedTransform); }
     }
+
+    pub fn set_scroll_snap_type_y(&mut self, v: longhands::scroll_snap_type_y::computed_value::T) {
```

Comment:
Why can't we make use of the existing Mako helper `impl_keyword`? It generates code for using enum variants and casting them to `u8` as necessary. (FYI, some of the existing properties already do this).